src/subpackages/core/cellprofiler_core/utilities/hdf5_dict.py

```diff
@@ -1457,13 +1459,15 @@ class HDF5ObjectSet(object):
 
     Segmentations are stored in one of two formats:
 
-    A 6-d array composed of one or more 5-d integer labelings of
+    A dense 6-d array composed of one or more 5-d integer labelings of
     each pixel. The dimension order is labeling, c, t, z, y, x. Typically,
     a 2-D non-overlapping segmentation has dimensions of 1, 1, 1, 1, y, x.
 
-    The i, j, v labeling of the pixels. The labeling is stored in a record
-    data type with each column having a name of "c", "t", "z", "y", "x" or
-    "label". The "label" column is the object number, starting with 1.
+    Sparse labeling stored in a record data type with each column having a name
+    of "c", "t", "z", "y", "x" or "label". The "label" column is the object
+    number, starting with 1. When "c", "t", "z" are absent, this is the same as
```

Comment:
"same as" is can be misconstrued. scipy does use ijv format, but wraps it in a special object. so should be "interoperable with" or "representable as", etc.